docs/code_overview.md

```diff
@@ -0,0 +1,109 @@
+# Code Overview
+
+## High-level Architecture
+
+The code base covers several user interfaces covering different aspects of the DUNE run
+control system. There are distinct interfaces but also shared infrastructure and
+approaches.
+
+For this reason the code is organised into three [Django apps]:
```

Comment:
```suggestion
For this reason the code is organised into the following [Django apps]:
```
So we can add more in the future minimising the inconsistencies when we - for sure - forget to update this "three" 😄